nethserver-subscription-inventory.spec

```diff
@@ -0,0 +1,44 @@
+%global debug_package %{nil}
+
+Summary: NethServer Subscriptions inventory agent
+Name: nethserver-subscription-inventory
+Version: 4.0.0
+Release: 1%{?dist}
+License: GPL
+URL: %{url_prefix}/nethserver-subscription
+Source0: %{name}-%{version}.tar.gz
+
+%ifarch x86_64
+Requires: puppet-agent
+%endif
+
+Provides: nethserver-inventory = %{version}
+Obsoletes: nethserver-inventory < %{version}
+
+%description
+NethServer Subscriptions inventory collects system facts and sends them every 
+day to a centralized server
+
+%prep
+%setup -q
+
+%build
+# noop
+
+%install
+install -m 0755 -D -T root/etc/cron.daily/nethserver-inventory %{buildroot}/etc/cron.daily/nethserver-inventory
```

Comment:
I think we should move here also this file: https://github.com/NethServer/nethserver-subscription/blob/master/root/etc/e-smith/events/actions/nethserver-inventory-send